recipes/pyramids/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "pyramids" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/MAfarrag/pyramids/archive/{{ version }}.tar.gz
+  sha256: c51393e7339fc70ef94bbdc683e82a0bb9c917eb38b75883e2ab16aec2cf9c8d
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - numpy >=1.21.2,<1.22.4
+    - gdal >=3.3.3,<3.5.1
+    - affine >=2.3.0
+    - fiona >=1.8.20
+    - pyproj >=3.2.1,<=3.4.0
+    - pandas >=1.3.2,<1.4.3
+    - geopandas >=0.10.2
+    - matplotlib >=3.4.2,<3.6.0
+    - shapely >=1.8.0
+    - rasterio >=1.2.10
+    - netCDF4 >=1.5.5,<1.5.9
+    - geopy >=2.2.0
+    - geoplot >=0.5.1
+    - pytest-cov >=2.12.1
+    - pytest >=6.2.5
+test:
+  imports:
+    - pyramids
+
+about:
+  home: https://github.com/MAfarrag/pyramids
+  license: GPL3
```

Comment:
Repeating myself a little bit, the `license` needs to be something found in `Identifier` column exactly:
![image](https://user-images.githubusercontent.com/90008/170835889-db4db479-625d-41b8-b323-7969e296aa8d.png)

https://spdx.org/licenses/